libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,21 @@
+<nav role="navigation" attr.aria-label="Pagination Navigation">
+  <a attr.aria-label="Go to Previous Page" tabindex="0" (click)="onNotifyPrevPageChange()" *ngIf="showPrevPageChange()" class="daff-paginator__prev">
+    <fa-icon [icon]="['fa', 'chevron-left']"></fa-icon>Prev
+  </a>
+
+  <span class="daff-paginator__ellipsis" *ngIf="showFirstEllipsis()"><a tabindex="0" attr.aria-label="Go to Page 1" class="daff-paginator__page-link" (click)="onNotifyPageChange(1)" [class.daff-paginator__page-link--selected]="isSelected(1)">1</a>...</span>
+  <span *ngFor="let pageNumber of _numberOfPagesArray">
+    <a tabindex="0" attr.aria-label="Go to Page {{pageNumber + 1}}"
```

Comment:
Why are all of these `+1`s littered about? Should this calculation be done in the template?

Reply:
yep fixed